app/controllers/issues_controller.rb

```diff
@@ -159,4 +171,16 @@ def tag_issue_from_field_content(issue)
       issue.save
     end
   end
+
+  def destroy_multiple_issues(issue_ids)
+    issue_ids.each do |id|
+      issue = Issue.find_by_id(id)
+      if issue
```

Comment:
In the singular destroy action we are also checking if the issue was destroyed. May be:
```
if issue && issue.destroy
```